pytest_fixtures/user_fixtures.py

```diff
@@ -20,3 +25,33 @@ def default_viewer_role(module_org, default_location):
     ).create()
     custom_user.password = custom_password
     return custom_user
+
+
+@pytest.fixture(scope='module')
+def module_user(request, module_org, module_loc):
+    """Creates admin user with default org set to module org and shares that
+    user for all tests in the same test module. User's login contains test
+    module name as a prefix.
+
+    :rtype: :class:`nailgun.entities.Organization`
+    """
+    # take only "module" from "tests.ui.test_module"
+    test_module_name = request.module.__name__.split('.')[-1].split('_', 1)[-1]
+    login = f'{test_module_name}_{gen_string("alphanumeric")}'
+    password = gen_string('alphanumeric')
+    LOGGER.debug('Creating session user %r', login)
+    user = entities.User(
+        admin=True,
+        default_organization=module_org,
+        default_location=module_loc,
+        description=f'created automatically by airgun for module "{test_module_name}"',
```

Comment:
This is created by nailgun. Or do you mean created for airgun?

Reply:
this is straight copied from the original function, so the original function was wrong in stating this was done via airgun

Reply:
ahhh ok. yeah unless i missed an import due to git context, this is nailgun

Reply:
Fixed